t/140-register-driver-after-cfg.t

```diff
@@ -35,6 +35,11 @@ local function check_driver_register()
     end
 
     test:ok(check_standart_drivers, 'standard drivers are defined')
+
+    local res, err = pcall(queue.register_driver, 'mock', mock_tube)
+    local check = not res and
+        string.match(err, 'overriding registered driver') ~= nil
+    test:ok(check, 'check a driver override failure')
```

Comment:
Maybe add the same test to `001-tube-init.t` to verify the same situation, when `register_driver()` is called before `box.cfg()`?

Reply:
Added.